$NetBSD: patch-ad,v 1.3 2002/03/09 21:48:12 fredb Exp $

Note to Porters
===============

This patch isn't actually used by the package, rather it's included for
the convenience of the packagers. If the subsequent patch to configure
fails, the patched configure.in can be used to regenerate configure.

--- configure.in.orig	Wed Apr 21 21:47:53 1999
+++ configure.in	Tue Sep 21 15:43:57 1999
@@ -64,6 +64,7 @@
 AC_AIX
 
 dnl Checks for libraries.
+AC_CHECK_LIB(db2, db_appinit)
 AC_CHECK_LIB(socket, socket)
 AC_CHECK_LIB(nsl, t_accept)
 if test $WITH_ZLIB = yes; then
@@ -140,7 +141,7 @@
 if test ! -d "./db"; then
   mkdir db
 fi
-AC_CONFIG_SUBDIRS(db/dist)
+dnl AC_CONFIG_SUBDIRS(db/dist)
 
 AC_OUTPUT(CONFIG Makefile Makefile.config htcommon/Makefile htlib/Makefile htdig/Makefile htmerge/Makefile htnotify/Makefile htfuzzy/Makefile htsearch/Makefile makedp, chmod +x makedp, echo timestamp > stamp-h)
 
